LayoutTests/fast/dom/Node/cloneChild-recursion-limit.html

```diff
@@ -0,0 +1,36 @@
+<html>
+    <head>
+        <title>Limit cloneChild Recursion Test</title>
+        <script>
+            if (window.testRunner) testRunner.dumpAsText();
```

Comment:
Nit: testRunner.dumpAsText should be on a separate line.
Alternatively, `window?.testRunner?.dumpAsText()` will do.